src/components/manage/Controlpanels/Users/RenderUsers.jsx

```diff
@@ -56,6 +67,40 @@ class RenderUsers extends Component {
     const [user, role] = value.split('.');
     this.props.updateUser(user, role);
   }
+
+  onEditUserSubmit(data, callback) {
+    // Do not handle groups and roles in this form
+    delete data.groups;
+    delete data.roles;
+    this.props.updateUserData(data.id, data);
+    this.setState({ user: {} });
+    this.props.listUsers();
+    return toast.success(
+      <Toast
+        success
+        title={this.props.intl.formatMessage(messages.success)}
+        content={this.props.intl.formatMessage(messages.updateUserSuccess)}
+      />,
+    );
+  }
```

Comment:
The change should be reflected after submit. Therefore the `listUsers` should be done after the update.
```suggestion
  onEditUserSubmit(data, callback) {
    // Do not handle groups and roles in this form
    delete data.groups;
    delete data.roles;
    this.props
      .updateUserData(data.id, data)
      .then((resp) => {
        this.setState({ user: {} });
        this.props.listUsers();
      })
      .then(() => {
        toast.success(
          <Toast
            success
            title={this.props.intl.formatMessage(messages.success)}
            content={this.props.intl.formatMessage(messages.updateUserSuccess)}
          />,
        );
      });
    return;
  }
```

Reply:
A nice to have would be: If a searchstring is given, then take it into account for `listUsers`.

Reply:
I managed to do it (I think) :wink:

Reply:
Great that you found a way. Glad you took the time to figure out. Merci! Works fine now.